airbyte-workers/src/main/java/io/airbyte/workers/internal/DefaultAirbyteSource.java

```diff
@@ -150,4 +156,17 @@ public void cancel() throws Exception {
     }
   }
 
+  private void logInitialStateAsJSON(final WorkerSourceConfig sourceConfig) {
+    if (!logConnectorMessages) {
+      return;
+    }
+
+    try {
+      final String json = new ObjectMapper().writeValueAsString(sourceConfig.getState().getState());
+      LOGGER.info("source starting state | " + json);
+    } catch (JsonProcessingException e) {
+      LOGGER.warn("Error serializing " + sourceConfig.getState() + " to JSON: " + e);
```

Comment:
should this also be `sourceConfig.getState().getState()` same as above?

Reply:
removed in https://github.com/airbytehq/airbyte/pull/15895/commits/3b0006f7347a7f6115a08d0784fc64345ae2bbc7